src/components/main-page/Index.tsx

```diff
@@ -13,6 +14,7 @@ export function Index() {
       <Hero />
       <EventDetails />
       <Registration />
+      <About />
```

Comment:
If we'd like to merge it to `master` we may want to comment that out for time being to not to expose that section, cause CD would built it and make available at https://code.akai.org.pl/